conf/nc48ads_a100_v4.conf

```diff
@@ -39,3 +39,4 @@
  * || check_gpu_ecc 20000000 10000
  * || check_gpu_clock_throttling
  * || check_nvlink_status
+ * || check_nccl_allreduce 22.5 1 $AZ_NHC_ROOT/topofiles/ncv4-topo.xml 8G
```

Comment:
Do we expect there to be a perf difference between nc48ads and nc96ads?

Reply:
Removed the test until we get more info on graph files